lib/ingram_micro/elements/sales_order_line_item.rb

```diff
@@ -1,37 +1,95 @@
-class IngramMicro::SalesOrderLineItem < IngramMicro::BaseElement
-
-  DEFAULTS = {
-    :line_no => nil,
-    :item_code => nil,
-    :universal_product_code => nil,
-    :product_name => nil,
-    :comments => nil,
-    :quantity => 1.0,
-    :unit_of_measure => nil,
-    :sid => nil,
-    :esn => nil,
-    :min => nil,
-    :mdn => nil,
-    :irdb => nil,
-    :imei => nil,
-    :market_id => nil,
-    :line_status => nil,
-    :base_price => 0.0,
-    :line_discount => 0.0,
-    :line_tax1 => 0.0,
-    :line_tax2 => 0.0,
-    :line_tax3 => 0.0
-  }
-
-  def line_no
-    @element[:line_no]
-  end
+module IngramMicro
+  class SalesOrderLineItem < BaseElement
 
-  def line_no=(num)
-    @element[:line_no] = num
-  end
+    DEFAULTS = {
+      line_no: nil,
+      item_code: nil,
+      universal_product_code: nil,
+      product_name: nil,
+      comments: nil,
+      quantity: 1.0,
+      unit_of_measure: nil,
+      sid: nil,
+      esn: nil,
+      min: nil,
+      mdn: nil,
+      irdb: nil,
+      imei: nil,
+      market_id: nil,
+      line_status: nil,
+      base_price: 0.0,
+      line_discount: 0.0,
+      line_tax1: 0.0,
+      line_tax2: 0.0,
+      line_tax3: 0.0
+    }.freeze
+
+    INTL_DEFAULTS = {
```

Comment:
agreed i think this can happen in all the files with `DEFAULTS` and `INTL_DEFAULTS`